server/utils/nunjucksSetup.ts

```diff
@@ -32,7 +32,7 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
     [
       path.join(__dirname, '../../server/views'),
       'node_modules/govuk-frontend/dist/',
-      'node_modules/govuk-frontend/dist/components/',
+      'node_modules/govuk-frontend/dist/govuk/components/',
```

Comment:
Should we also add it to line 34 above?

Reply:
no because then import paths from the docs (e.g. `{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}`) would not work

Reply:
I'm not too sure how this works to be honest/why we need a higher-level directory and a lower-level one. This is roughly in line with the lines below though:

```ts
      'node_modules/@ministryofjustice/frontend/',
      'node_modules/@ministryofjustice/frontend/moj/components/',
```

Reply:
in fact, what i do in our team's projects is only retain these 2 search paths so that there is only one consistent route to each template
• `'node_modules/govuk-frontend/dist/'`
• `'node_modules/@ministryofjustice/frontend/'`

Reply:
@ushkarev @yndajas happy to go with ☝️

Reply:
I've pushed an update and credited you as co-author @ushkarev - hope that's okay!

Reply:
whoa whoa don't blame me :p